pkg/tfbridge/info.go

```diff
@@ -96,6 +96,13 @@ type ProviderInfo struct {
 	PreConfigureCallback           PreConfigureCallback // a provider-specific callback to invoke prior to TF Configure
 	PreConfigureCallbackWithLogger PreConfigureCallbackWithLogger
 
+	// Information for the embedded metadata file.
+	//
+	// If provided, the user should call NewMetadataInfo(path, bytes) where path is
```

Comment:
The zero value works great (which is why the type is `*MetadataInfo`). I'll fix the language.

Reply:
Ahh I see. I meant something like MetadataInfo{bytes: ..} forgetting the path, which would default to something. I see you have some validation in New so that's fine too.